src/js/components/AppPageComponent.jsx

```diff
@@ -15,27 +23,23 @@ var tabsTemplate = [
   {id: "apps/:appid/configuration", text: "Configuration"}
 ];
 
+var statusNameMapping = {};
+statusNameMapping[AppStatus.RUNNING] = "Running";
+statusNameMapping[AppStatus.DEPLOYING] = "Deploying";
+statusNameMapping[AppStatus.SUSPENDED] = "Suspended";
+
```

Comment:
This is great! I've tried it and it works with Babel! :)

Unfortunately it doesn't work with our test compiler what uses the React tools.
We should take a look into this separately.